platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/log/Logger.java

```diff
@@ -221,4 +262,60 @@ public static void log(int severity, String tag, String message) {
         throw new UnsupportedOperationException();
     }
   }
+
+  /**
+   * Priority constant for the println method; use Logger.v
+   * <p>
+   * This log level will print all logs.
+   * </p>
+   */
+  public static final int VERBOSE = Log.VERBOSE;
+
+  /**
+   * Priority constant for the println method; use Logger.d.
+   * <p>
+   * This log level will print all logs except verbose.
+   * </p>
+   */
+  public static final int DEBUG = Log.DEBUG;
+
+  /**
+   * Priority constant for the println method; use Logger.i.
+   * <p>
+   * This log level will print all logs except verbose and debug.
```

Comment:
If I'm reading this and https://github.com/mapbox/mapbox-gl-native/blob/f3341dd589c6330c0cfd6e8e381398d08493b48a/src/mbgl/util/event.cpp correctly, right now setting this api to `VERBOSE` and `DEBUG` would print the same logs since we don't have `VERBOSE` as a severity classification in our logging.

@tmpsantos, should we add `VERBOSE` as a level that we may want to use in the future?

cc @tobrun @LukasPaczos @julianrex

Reply:
if there is not reason to add VERBOSE to core, we shouldn't. These loglevels come from https://developer.android.com/reference/android/util/Log. The Logger class is used for more things as just logging out core logs.